### DISCUSSION TITLE
Obtaining delta B estimates

### DISCUSSION THREAD

**Discussion by juanitagutierrez** [LohseLab/gIMble#141] [Q&A] [answered]
Hi again,

I don't know if this is the correct place to drop this question, but just in case ;)
I obtained _me_ estimates using `gimble gridsearch`. I understand these could be used directly to identify loci with reduced effective migration, but I would like to obtain delta B to account for local variation in Ne. If I understand formula 1 in Laetsch, Bisschop, et al. (2023) correctly, one would obtain such estimate per loci by subtracting each lnCl value from the lnCl of the best global model? 

Thanks in advance for your answer!

Best,

**Comment by GertjanBisschop** [comment, marked as answer]
That's right. (I assume you mean window where it says locus).

**Comment by MengLu-flw** [comment]
Hi both!

Thank you so much for opening this thread & I am really interested in this topic (how to obtain ΔB) too!
Sorry to add more comments if the discussion regarding this topic has closed.

Based on two previous posts, it seems that ΔB (at window_i) = lnCL (the log composite likelihood of window_i, obtained from ‘gimble makegrid’ > ‘gimble gridsearch’ > ‘gimble query’ under the best global model) - lnCL (the log composite likelihood of the best global model, a single defined value). Please correct me if my understanding of your discussion is wrong.

However, my understanding of the Equation (1) in Laetsch, Bisschop, et al. (2023) is:

If IM model is the best global model, then at ‘gimble makegrid’ step, fix T and allow Ne (C, A, B) and M (me) to be re-estimated per window, based on the empirical data.

To obtain the ΔB, apart from running ‘makegrid’ & ‘gridsearch’ with fixed T under the best fitting model (described above), we do an independent run - still under the best fitting model, but fixed both T and M (me) to be the same as the global best estimates. Let Ne (C, A, B) be re-estimated per window.

Then, ΔB (at window_i) = lnCL (the log composite likelihood of window_i, the best IM model, fixed T) - lnCL (the log composite likelihood of window_i, the best IM model, fixed T and M).

Hope I explained my understanding clearly and please let me know if my understanding is correct or not.
Many thanks in advance :^)

Gratefully,
Meng
